inc/log.class.php

```diff
@@ -704,6 +823,367 @@ static function getHistoryData(CommonDBTM $item, $start = 0, $limit = 0, array $
       return $changes;
    }
 
+   /**
+    * Retrieve distinct values for user_name field in item log.
```

Comment:
missing since

Reply:
Fixed in commit f34462212c26558b8c52df65266ef7db60458c7e.